src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The excavator started working");

```

Comment:
The message in the `doWork` method should indicate what type of `Machine` is working. Consider specifying that it is an `Excavator` in the message, for example: 'Excavator started its work.'